coordinator/src/main/java/org/apache/uniffle/coordinator/strategy/storage/LowestIOSampleCostSelectStorageStrategy.java

```diff
@@ -99,26 +100,38 @@ public void sortPathByRankValue(
   public void detectStorage() {
```

Comment:
Could you merge `detectStorage` of `LowestIOSampleCostSelectStorageStrategy` and `AppBalanceSelectStorageStrategy`.
The logic is almost the same.

Reply:
Updated